### PR TITLE
Adds info on preinstalled hardware

### LINKED ISSUE #515: Document "btcpay in a box" devices
There are lots of people selling pre-built btcpya server on a certain type of hardware. 

It would be good if we can have a document listing all the productsthat offer it. Not everyone wants to build their own, some may want a plug and play solution. 

We touched on this in HardwareDeployment.md but def can be a separate document.

On top of my head:
- [Nodl](https://www.nodl.it/)
- [Hack0](https://www.dglab.com/en/works/hack0/)
- [Lightning in a box](https://lightninginabox.co/)
- ~~Casa 2~~ (actually I don't think they ever released it)
- [MyNode](https://mynodebtc.com/)
- [RaspiBlitz ](https://raspiblitz.com/)

I am unsure of the name of this doc. It can probably be nested under hardware deployment once our new docs are ready.

### PR DESCRIPTION
**Pull request by Zaxounette**
This draft adds info on preinstalled hardware boxes.
Relates to #515

I named the document and the title "Hardware As A Service" because I liked the sound of it.
Name suggestions appreciated.
DIYNodes for example.

I will commit the summary change in config.js once this PR is merge-able. Possibly just under "Hardware Deployment", but not as a child.

EDIT AFTER COMMENTS:

Alright so: 
* I added HardwareAsAService.md to config.js as a child of Hardware Deployment.
* No objections on the name "Hardware As A Service".
* Pavlenex, you stated that Hardware Deployment needs a workup, so this can just be a temporary add until the restructure is done.
* Closes #515

### CODE REVIEW

**Comment by britttttk** [comment]
I like the idea @Zaxounette .The name makes sense to me. There is also casa with btcpay.

File: docs/HardwareAsAService.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+Multiple entities or companies in the Bitcoin space provide preinstalled hardware that include BTCPay Server.
+Installed for example on Raspberry Pi or Rock64 devices, the BTCPay Server instance sits besides other software stacks.
+
+This Hardware As A Service can be free or paid, and is mostly open-sourced for peer-review.
+
+They are suitable for technical users to speed up the installation and intitial syncing process.
+Less-technical users will find the 1-click or plug-and-play systems of these services incredibly practical.
+
+Here you can find a non-exhaustive list of companies that provide such a service:
+- [Nodle](https://www.nodl.it/)
```
Comment (Kukks) [inline docs/HardwareAsAService.md:12]:
```suggestion
- [Nodl](https://www.nodl.it/)
```

File: docs/HardwareAsAService.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+- [Hack0](https://www.dglab.com/en/works/hack0/)
+- [LightningInABox](https://lightninginabox.co/)
+- [MyNode](https://mynodebtc.com/)
+- [RaspyBlitz](https://shop.fulmo.org/raspiblitz/)
```
Comment (Kukks) [inline docs/HardwareAsAService.md:16]:
```suggestion
- [RaspiBlitz](https://shop.fulmo.org/raspiblitz/)
```

File: docs/HardwareAsAService.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+This Hardware As A Service can be free or paid, and is mostly open-sourced for peer-review.
+
+They are suitable for technical users to speed up the installation and intitial syncing process.
+Less-technical users will find the 1-click or plug-and-play systems of these services incredibly practical.
+
+Here you can find a non-exhaustive list of companies that provide such a service:
+- [Nodle](https://www.nodl.it/)
+- [Hack0](https://www.dglab.com/en/works/hack0/)
+- [LightningInABox](https://lightninginabox.co/)
```
Comment (Kukks) [inline docs/HardwareAsAService.md:14]:
```suggestion
- [LightningInABox](https://lightninginabox.co/)
- [LuxNode](https://luxnode.io/)
```

**Comment by Zaxounette** [comment]
> Are you sure this shouldn't just be a link to a category in the directory?

I proposed as such to @pavlenex in #515 , and the answer ( [here](https://github.com/btcpayserver/btcpayserver-doc/issues/515#issuecomment-628057860) ) was that it would be more usefull in the documentation, so I went with it.

This being a draft, I can delete everything and just make a new directory cat and add them to it, it's not a problem :)

**Comment by Zaxounette** [comment]
> I like the idea @Zaxounette .The name makes sense to me. There is also casa with btcpay.

The Casa 2 that includes BTCPay Server is sold out, and I don't think they'll reiterate ?

Also, I don't think it's open sourced yet. Would that be a nono for anyone as to including them in this list ?

**Comment by pavlenex** [comment]
Casa 2 never had BTCPay afaik, it was planned but they quickly dropped their device.

**Comment by Zaxounette** [comment]
> Casa 2 never had BTCPay afaik, it was planned but they quickly dropped their device.

Okay, so no Casa 2 then.

**Comment by Zaxounette** [comment]
@pavlenex Was this the kind of info you wanted in this doc ?
Also, documentation or directory for the list ?

**Comment by pavlenex** [comment]
@Zaxounette  Yes, I'll have to think about the structure and where we can put this, to me docs make more sense as I already mentioned. I'd add this in a hardware deployment section, but that section needs restructure as well.

File: docs/.vuepress/config.js [hunk 4]
```diff
@@ -136,6 +136,10 @@ module.exports = {
                       ["/ReverseSSHtunnel", "Reverse SSH Tunnel"]
                     ]
                   }
+				  {
+					title: "Hardware As A Service"
+					path: "/HardwareAsAService"
+				  }
```
Comment (Zaxounette) [inline docs/.vuepress/config.js:142]:
```suggestion
                  }
			title: "Hardware As A Service"
			path: "/HardwareAsAService"
                  }
```

File: docs/.vuepress/config.js [hunk 5]
```diff
@@ -136,6 +136,10 @@ module.exports = {
                       ["/ReverseSSHtunnel", "Reverse SSH Tunnel"]
                     ]
                   }
```
Comment (dennisreimann) [inline docs/.vuepress/config.js:138]: The indentation is awry (we can fix it later on), but this line is missing a trailing comma, which should lead to a build error.

Comment (Zaxounette) [reply docs/.vuepress/config.js:138]: It just did yeah.

Was also fixing the spacing problem that I still have.

Comment (Zaxounette) [reply docs/.vuepress/config.js]: Still failing check.

**Comment by Zaxounette** [comment]
Alright so:

* I added HardwareAsAService.md to config.js as a child of Hardware Deployment.
* No objections on the name "Hardware As A Service".
* Pavlenex, you stated that Hardware Deployment needs a workup, so this can just be a temporary add until the restructure is done.
* Closes issue 515

Checklink failed though. @dennisreimann ?

**Comment by dennisreimann** [comment]
The linkcheck looks like a temporary glitch, the [link](https://www.reddit.com/r/Bitcoin/comments/8f1eqf/the_ultimate_guide_to_btcpay_the_free_and/) works for me.

File: docs/.vuepress/config.js [hunk 6]
```diff
@@ -135,7 +135,11 @@ module.exports = {
                       ["/DynamicDNS", "Dynamic DNS"],
                       ["/ReverseSSHtunnel", "Reverse SSH Tunnel"]
                     ]
-                  }
+                  },
+		 							{
+		    						title: "Hardware As A Service",
+		    						path: "/HardwareAsAService"
+		  						}
```
Comment (dennisreimann) [inline docs/.vuepress/config.js:142]:
```suggestion
                  },
                  {
                    title: "Hardware As A Service",
                    path: "/HardwareAsAService"
                  }
```

**Comment by ketominer** [comment]
> I like the idea @Zaxounette .The name makes sense to me. There is also casa with btcpay.

it's discontinued ;)

**Pull request merged**